docker/1.13/Dockerfile.cpu

```diff
@@ -1,29 +1,75 @@
-ARG TFS_VERSION
-
-FROM tensorflow/serving:${TFS_VERSION} as tfs
 FROM ubuntu:18.04
-LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
-COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 
 # nginx + njs
-RUN \
-    apt-get update && \
-    apt-get -y install --no-install-recommends curl gnupg2 && \
-    curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - && \
-    # find the correct linux distribution here - http://nginx.org/en/linux_packages.html
-    echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools && \
-    apt-get clean
-
-# cython, falcon, gunicorn, tensorflow-serving
-RUN \
-    pip3 install --no-cache-dir cython falcon gunicorn gevent requests grpcio protobuf tensorflow && \
-    pip3 install --no-dependencies --no-cache-dir tensorflow-serving-api
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends curl gnupg2 ca-certificates git wget vim build-essential zlib1g-dev \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools \
+ && apt-get clean
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+ARG PYTHON=python3
+ARG PIP=pip3
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1
+
+# cython, falcon, gunicorn
+RUN ${PIP} install \
+    cython==0.29.10 \
+    falcon==2.0.0 \
+    gunicorn==19.9.0 \
+    gevent==1.4.0 \
+    requests==2.21.0
 
 COPY ./ /
 
-ARG TFS_SHORT_VERSION
+ARG TFS_SHORT_VERSION=1.13
```

Comment:
lets move all ARGS and ENV to top of file

Reply:
Done